### PR TITLE
adds config for using a minimal camunda and a locally configured kubernetes 

### PR DESCRIPTION
**Pull request by nyuuyn**

### CODE REVIEW

**Comment by options-a11y** [comment]
I think you need to adjust the .gitignore file, as the folder structure has changed for the tf scripts.

File: README.md [hunk 1]
```diff
@@ -18,15 +19,27 @@ All test ressources are under `src/` and `tf/`. while the first folder contains
...
-  - Camunda Zeebee Gateway via: ``kubectl port-forward -n camunda8 svc/camunda-platform-zeebe-gateway 26500:26500`` and afterwards via ``localhost:26500`` and creds ``demo:demo``
+- Setting up the environments
+  - Running C8 on the locally configured kubernetes cluster i.e. the cluster which is active via the ``.kubeconf`` file
+    - Use ``terraform init`` and ``terraform apply`` in the ``tf/c8`` of the project
+  - (Optional) If you want to use an AWS EKS cluster configured according to Camunda and setup your ``.kubeconf`` accordingly
+    - Use ``terraform init`` and ``terraform apply`` in the ``tf/eks`` of the project and afterwards setup C8
+- (Camunda 8) When everything is finished you can access the following
+  - (🗡️ Important 🗡️) If you have Camunda Identity and Keycloak enabled you should keep the ports of these open via:
+    - ``kubectl port-forward -n camunda svc/camunda-platform-keycloak 18080:8080`` and ``kubectl port-forward -n camunda svc/camunda-platform-identity 8080:8080`` afterwards via ``localhost:8083`` and creds ``demo:demo``
```
Comment (options-a11y) [inline README.md]:
```suggestion
    - ``kubectl port-forward -n camunda svc/camunda-platform-keycloak 18080:80`` and ``kubectl port-forward -n camunda svc/camunda-platform-identity 8080:80`` and afterwards via ``localhost:8080`` and creds ``demo:demo``
```

File: README.md [hunk 2]
```diff
@@ -18,15 +19,27 @@ All test ressources are under `src/` and `tf/`. while the first folder contains
...
+- Setting up the environments
+  - Running C8 on the locally configured kubernetes cluster i.e. the cluster which is active via the ``.kubeconf`` file
+    - Use ``terraform init`` and ``terraform apply`` in the ``tf/c8`` of the project
+  - (Optional) If you want to use an AWS EKS cluster configured according to Camunda and setup your ``.kubeconf`` accordingly
+    - Use ``terraform init`` and ``terraform apply`` in the ``tf/eks`` of the project and afterwards setup C8
+- (Camunda 8) When everything is finished you can access the following
+  - (🗡️ Important 🗡️) If you have Camunda Identity and Keycloak enabled you should keep the ports of these open via:
+    - ``kubectl port-forward -n camunda svc/camunda-platform-keycloak 18080:8080`` and ``kubectl port-forward -n camunda svc/camunda-platform-identity 8080:8080`` afterwards via ``localhost:8083`` and creds ``demo:demo`` 
+  - Camunda Operate via: 
+    - ``kubectl port-forward -n camunda svc/camunda-platform-operate  8080:80`` and afterwards via ``localhost:8080`` and creds ``demo:demo``
```
Comment (options-a11y) [inline README.md]:
```suggestion
    - ``kubectl port-forward -n camunda svc/camunda-platform-operate  8081:80`` and afterwards via ``localhost:8081`` and creds ``demo:demo``
```

File: README.md [hunk 3]
```diff
@@ -18,15 +19,27 @@ All test ressources are under `src/` and `tf/`. while the first folder contains
...
+  - Camunda Tasklist via: 
+    - ``kubectl port-forward -n camunda svc/camunda-platform-tasklist  8081:80`` and afterwards via ``localhost:8081`` and creds ``demo:demo``
```
Comment (options-a11y) [inline README.md]:
```suggestion
    - ``kubectl port-forward -n camunda svc/camunda-platform-tasklist  8082:80`` and afterwards via ``localhost:8082`` and creds ``demo:demo``
```

**Comment by CRoberto1926** [comment]
You can probably link this PR to this issue: https://github.com/envite-consulting/CarboNata/issues/9

**Comment by CRoberto1926** [comment]
We should consolidate the readme with https://docs.camunda.io/docs/self-managed/setup/guides/accessing-components-without-ingress/

**Review by lwluc — APPROVED** [review]

**Pull request merged**